coremltools/converters/mil/frontend/tensorflow/test/test_ops.py

```diff
@@ -6726,3 +6726,32 @@ def build_model(x):
             frontend_only=False,
             backend=backend,
         )
+
+
+class TestReduceSum(TensorFlowBaseTest):
+    @pytest.mark.parametrize(
+        "use_cpu_only, backend, input_dtype",
+        itertools.product(
+            [True, False],
+            backends,
+            [tf.float32, tf.int32]  # tf.reduce_sum doesn't support bool input.
+        ),
+    )
+    def test_reduce_sum(self, use_cpu_only, backend, input_dtype):
```

Comment:
tf.reduce sum is already being tested in
`class TestReduction(TensorFlowBaseTest)`
Any reason creating this additional test beside the `tf.int32` input type?

Reply:
Thank you for the info! Removed the test case.